README.md

```diff
@@ -1,229 +1,67 @@
-```
+:eight_spoked_asterisk: [Get started with AzCopy](https://docs.microsoft.com/azure/storage/common/storage-use-azcopy-v10)
 
-### Advanced Use Cases
+:eight_spoked_asterisk: [Transfer data with AzCopy and blob storage](storage-use-azcopy-blobs.md)
 
-#### Configure Concurrency
+:eight_spoked_asterisk: [Transfer data with AzCopy and file storage](storage-use-azcopy-files.md)
 
-Set the environment variable `AZCOPY_CONCURRENCY_VALUE` to configure the number of concurrent requests. This is set to 300 by default. Note that this does not equal to 300 parallel connections. Reducing this will limit the bandwidth, and CPU used by AzCopy.
+:eight_spoked_asterisk: [Transfer data with AzCopy and Amazon S3 buckets](storage-use-azcopy-s3.md)
 
-#### Configure proxy settings
-To configure the proxy settings for AzCopy v10, set the environment variable https_proxy using the following command:
+:eight_spoked_asterisk: [Configure, optimize, and troubleshoot AzCopy](storage-use-azcopy-configure.md)
 
-```
-# For Windows:
-set https_proxy=<proxy IP>:<proxy port>
-# For Linux:
-export https_proxy=<proxy IP>:<proxy port>
-# For MacOS
-export https_proxy=<proxy IP>:<proxy port>
-```
+### Find help from your command prompt
 
-For proxy authentication, use the following format:
+Be sure to add AzCopy to your system path. Then, open a command prompt window.
 
-```
-export https_proxy=<user>:<pass>@<proxy IP>:<proxy port>
-# or with a domain:
-export https_proxy=<domain>%5C<user>:<pass>@<proxy IP>:<proxy port>
-```
+To see a list of commands, type `azcopy` and then press the ENTER key.
 
-### Configure log location
+To learn about a specific command, type `azCopy` followed by the name of the command.
 
-Set the environment variable 'AZCOPY_LOG_LOCATION' to a directory of your choice where there is plenty of disk space as logs for large data transfers may use up Gigabytes of space depending on the chosen log level.
+![AzCopy command help example](readme-command-prompt.png)
 
-## Troubleshooting and Reporting Issues
+## Frequently asked questions
 
-### Check Logs for errors
+### What is the difference between `sync` and `copy`?
 
-AzCopy creates a log file for all the jobs. Look for clues in the logs to understand the problem. AzCopy will print UPLOADFAILED, COPYFAILED, and DOWNLOADFAILED strings for failures with the paths along with the error reason.
+The `copy` command is a simple transferring operation, it scans the source and attempts to transfer every single file/blob. The supported source/destination pairs are listed in the help message of the tool. On the other hand, `sync` makes sure that whatever is present in the source will be replicated to the destination, and also whatever is not at the source will be deleted from the destination. If your goal is to simply move some files, then `copy` is definitely the right command, since it offers much better performance.
```

Comment:
Perhaps it's better to clarify that by default sync leaves the extra files at the destination alone. The user can choose to delete them.

Reply:
Yes. Maybe use a followup section for this (similar to the existing one about overwrite).  the sync one could be title "Will sync delete files from the destination?" and would explain the relevant paramter.

---
In reply to: [289576263](https://github.com/Azure/azure-storage-azcopy/pull/426#discussion_r289576263) [](ancestors = 289576263)

Reply:
Ok Hows this?